make/RunTests.gmk

```diff
@@ -356,6 +374,7 @@ ExpandJtregPath = \
 # specified test root, or a path to a test or test directory,
 # either absolute or relative to any of the TEST_BASEDIRS or test roots.
 define ParseJtregTestSelection
+$(call IfAppend, \
```

Comment:
Please adjust indentation of the whole body here.

Reply:
Fixed